sdk/python/feast/feature_view.py

```diff
@@ -199,6 +199,25 @@ def is_valid(self):
         if not self.entities:
             raise ValueError("Feature view has no entities.")
 
+    def with_name(self, name: str):
+        """
+        Produces a copy of this FeatureView with the passed name.
```

Comment:
Can we add `name` as an argument in the docstring?

Reply:
yep, will add documentation for the argument. We'll add an example in this docstring in the shadow entities mapping change cc: @codyjlin .